src/cli/index.js

```diff
@@ -50,6 +51,14 @@ export default class Cli {
   async run() {
     await validateInstall(this.cwd)
     const configuration = await this.getConfiguration()
+    if (configuration.listI18nLanguages) {
```

Comment:
I think the current interface is simpler to unit test (not that I added unit tests, but just for future) since the methods just return strings. If we kept the current interface, should we rename the methods? Maybe just make them get* methods?